src/components/about-chat-sidebar/AboutChatSidebar.tsx

```diff
@@ -0,0 +1,144 @@
+import ImageOutlinedIcon from '@mui/icons-material/ImageOutlined'
+import InsertDriveFileOutlinedIcon from '@mui/icons-material/InsertDriveFileOutlined'
+import LinkOutlinedIcon from '@mui/icons-material/LinkOutlined'
+import Close from '@mui/icons-material/Close'
+
+import AppButton from '~/components/app-button/AppButton'
+import SidebarContentBox from '~/components/sidebar-content-box/SidebarContentBox'
+import SidebarImageGrid from '~/components/sidebar-image-grid/SidebarImageGrid'
+import FileComponent from '~/components/file-component/FileComponent'
+import LinkComponent from '~/components/link-component/LinkComponent'
+
+import { SizeEnum, ButtonVariantEnum, UserResponse, Link, File } from '~/types'
+import { createUrlPath, spliceSx } from '~/utils/helper-functions'
+import { authRoutes } from '~/router/constants/authRoutes'
+
+import { styles } from '~/components/about-chat-sidebar/AboutChatSidebar.styles'
+
+interface AboutChatSidebarProps {
+  user: UserResponse
+  media: Array<string>
+  files: File[]
+  links: Link[]
+}
+
+const AboutChatSidebar: FC<AboutChatSidebarProps> = ({
+  user,
+  media,
+  files,
+  links
+}) => {
+  const { t } = useTranslation()
+  const navigate = useNavigate()
+  const [isOpened, setIsOpened] = useState(true)
+
+  const {
+    _id,
+    firstName,
+    lastName,
+    photo,
+    role,
+    professionalSummary: userDescription
+  } = user
+
+  const userProfileURL = createUrlPath(authRoutes.userProfile.path, _id, {
+    role
+  })
+
+  const closeSidebar = () => {
+    setIsOpened(false)
+  }
+
+  return (
+    <Box
+      data-testid='sidebar'
+      style={isOpened ? undefined : { display: 'none' }}
+      sx={styles.wrapper}
+    >
+      <Box sx={styles.header}>
+        <Typography sx={spliceSx(styles.header.text, styles.title)}>
+          {t(`chat.sidebar.about`)}
+        </Typography>
+        <IconButton
+          aria-label='close'
+          onClick={closeSidebar}
+          sx={styles.header.iconButton}
+        >
+          <Close />
+        </IconButton>
+      </Box>
+      <Divider />
+      <Box sx={styles.chatInfo}>
+        <Avatar src={photo} sx={styles.chatInfo.avatar} />
+        <Typography sx={styles.title}>{`${firstName} ${lastName}`}</Typography>
+        <AppButton
+          onClick={() => navigate(userProfileURL)}
+          size={SizeEnum.Medium}
+          sx={styles.secondaryText}
+          variant={ButtonVariantEnum.Tonal}
+        >
+          View Profile
+        </AppButton>
+        <Typography sx={styles.chatInfo.description}>
+          {userDescription || t(`chat.sidebar.noSummary`)}
+        </Typography>
+      </Box>
+      <Divider />
+      <SidebarContentBox Icon={ImageOutlinedIcon} name={'Media'}>
+        {media.length !== 0 ? (
+          <SidebarImageGrid images={media} />
+        ) : (
+          <Typography sx={styles.notFound}>
+            {t(`chat.sidebar.noMedia`)}
+          </Typography>
+        )}
```

Comment:
better move this to variable: 

   const variableName = media.length !== 0 ? (
          <SidebarImageGrid images={media} />
        ) : (
          <Typography sx={styles.notFound}>
            {t(`chat.sidebar.noMedia`)}
          </Typography>
        )

Reply:
> better move this to variable:
> 
> const variableName = media.length !== 0 ? ( ) : ( {t(`chat.sidebar.noMedia`)} )

fixed